webui/react/Makefile

```diff
@@ -48,6 +49,9 @@ check-css:
 .PHONY: check-package-lock
 check-package-lock:
 	if grep 'ssh://' package-lock.json ; then echo "ssh url in package-lock.json, please convert to https url" ; false ; fi
+.PHONY: check-requirements
+check-requirements:
+	node --version | grep -E 'v16\.1[3-6]' || (echo "node version >=16.13 is required" ; false)
```

Comment:
maybe also print out the upper cap? <17

Reply:
doh just saw this. Here's a [follow up PR](https://github.com/determined-ai/determined/pull/4290) :D